suite-native/graph/src/components/TimeSwitch.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box } from '@suite-native/atoms';
+
+import { TimeFrameItems } from '../reducers/types';
+import { TimeSwitchItem } from './TimeSwitchItem';
+
+const timeSwitchItems: TimeFrameItems = {
+    hour: {
+        shortcut: '1h',
+        value: 'hour',
+    },
+    day: {
+        shortcut: '1d',
+        value: 'day',
+    },
+    week: {
+        shortcut: '1w',
+        value: 'week',
+    },
+    month: {
+        shortcut: '1m',
+        value: 'month',
+    },
+    year: {
+        shortcut: '1y',
+        value: 'year',
+    },
+    all: {
+        shortcut: 'all',
+        value: 'all',
+    },
+};
+
+const timeSwitchStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+}));
+
+export const TimeSwitch = () => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box style={applyStyle(timeSwitchStyle)}>
+            {Object.values(timeSwitchItems).map(item => (
```

Comment:
Could be memoized?

Reply:
Good point :)  https://github.com/trezor/trezor-suite/pull/5838/commits/8d03ac13bbdc855f35c585b400951d9764b552cb

Reply:
Please don't do that, cost of memoization is probably higher that cost of renredering. This component should be nearly never rerendered because it has no props, no internal or redux state... even if yes, cost rerendering that few items here will be very very small.

I am not saying this could be actually faster, but there is golden rule for doing optimalizations and that rule is: Don't do optimalizations without proper measurement, because you can make things worse than it was before. No optimilazation is better than bad and blindly done optimalization.

Reply:
You are right, it's just when there is a `.map()` in the render memo usually should save some renders. That component is not big and doesn't even have props, yes. I